doc/user/content/manage/troubleshooting.md

```diff
@@ -89,25 +321,34 @@ take up most of Materialize's memory use. Arrangements maintain indexes for data
 as it changes. These queries extract the numbers of records and
 batches backing each of the arrangements. The reported records may
 exceed the number of logical records; the report reflects the uncompacted
-state. The number of batches should be logarithmic-ish in this
-number, and anything significantly larger is probably a bug.
+state.
 
 ```sql
--- Extract arrangement records and batches, summed across workers.
-SELECT mdo.id, mdo.name, mas.records, mas.batches
+-- Extract arrangement records and batches
```

Comment:
Just a note that in 2 weeks we'll be exposing actual bytes used by arrangement!

Reply:
This is actually live in prod as of literally now :)
```
materialize=> select * from mz_internal.mz_arrangement_sizes;
 operator_id | records | batches |    size    |  capacity  | allocations
-------------+---------+---------+------------+------------+-------------
 372         |       0 |       1 |         16 |         16 |           7
 770         |      43 |       2 |       4397 |       4397 |          18
 364         |    9179 |       2 |    1028452 |    1028452 |          18
 368         |    9179 |       2 |    1028452 |    1028452 |          18
 755         | 1188533 |       2 |  128877811 |  128877963 |          18
 740         | 1768240 |       2 | 1561397696 | 1561397696 |          18
 802         | 5755315 |       2 | 1391255464 | 1391262526 |          47
 817         | 5755329 |       4 | 1718268926 | 1718280973 |          36
 725         | 5755303 |       3 | 1299892208 | 1299897269 |          18
(9 rows)
```